l1-contracts/contracts/bridgehub/IInteropCenter.sol

```diff
@@ -0,0 +1,74 @@
+// SPDX-License-Identifier: MIT
+// We use a floating point pragma here so it can be used within other projects that interact with the ZKsync ecosystem without using our exact pragma version.
+pragma solidity ^0.8.21;
+
+import {L2Message, L2Log, TxStatus} from "../common/Messaging.sol";
+import {IL1AssetHandler} from "../bridge/interfaces/IL1AssetHandler.sol";
+import {ICTMDeploymentTracker} from "./ICTMDeploymentTracker.sol";
+import {IMessageRoot} from "./IMessageRoot.sol";
+import {IAssetHandler} from "../bridge/interfaces/IAssetHandler.sol";
+import {L2TransactionRequestDirect, L2TransactionRequestTwoBridgesOuter} from "./IBridgehub.sol";
+
+/// @author Matter Labs
+/// @custom:security-contact security@matterlabs.dev
+interface IInteropCenter {
+    function setAddresses(address assetRouter) external;
+    /// Mailbox forwarder
+
+    function proveL2MessageInclusion(
+        uint256 _chainId,
+        uint256 _batchNumber,
+        uint256 _index,
+        L2Message calldata _message,
+        bytes32[] calldata _proof
+    ) external view returns (bool);
+
+    function proveL2LogInclusion(
+        uint256 _chainId,
+        uint256 _batchNumber,
+        uint256 _index,
+        L2Log memory _log,
+        bytes32[] calldata _proof
+    ) external view returns (bool);
+
+    function proveL1ToL2TransactionStatus(
```

Comment:
We have the same function on Bridgehub, what is our roadmap here? 

BTW, I am totally fine with having both in the current release but strongly announce that in the next upgrade they will be removed from the code. But in this case, please add easily visible comments in Bridgehub that tell that these methods should not be used